include/remill/BC/SleighLifter.h

```diff
@@ -74,7 +75,8 @@ class SleighLifter : public InstructionLifter {
   std::pair<LiftStatus, std::optional<llvm::Function *>>
   LiftIntoInternalBlockWithSleighState(
       Instruction &inst, llvm::Module *target_mod, bool is_delayed,
-      const std::optional<sleigh::BranchTakenVar> &btaken);
+      const std::optional<sleigh::BranchTakenVar> &btaken,
```

Comment:
I would type alias `std::optional< sleigh::BranchTakenVar >` as it is used often and is pretty long.

Reply:
[501636a](https://github.com/lifting-bits/remill/pull/645/commits/501636ae96dc4961ee46e8e1114954a18bc9f6a5)